community/OM-Operator-Management/cnv-samples/cnv-operator-install/policy-cnv.yaml

```diff
@@ -0,0 +1,179 @@
+                    disableSerialConsoleLog: true
+                  higherWorkloadDensity:
+                    memoryOvercommitPercentage: 100
+                  liveMigrationConfig:
+                    allowAutoConverge: false
+                    allowPostCopy: false
+                    completionTimeoutPerGiB: 800
+                    parallelMigrationsPerCluster: 5
+                    parallelOutboundMigrationsPerNode: 2
+                    progressTimeout: 150
+                  certConfig:
+                    ca:
+                      duration: 48h0m0s
+                      renewBefore: 24h0m0s
+                    server:
+                      duration: 24h0m0s
+                      renewBefore: 12h0m0s
+                  applicationAwareConfig:
+                    allowApplicationAwareClusterResourceQuota: false
+                    vmiCalcConfigName: DedicatedVirtualResources
+                  featureGates:
+                    deployTektonTaskResources: false
+                    enableCommonBootImageImport: true
+                    withHostPassthroughCPU: false
+                    downwardMetrics: false
+                    disableMDevConfiguration: false
+                    enableApplicationAwareQuota: false
+                    deployKubeSecondaryDNS: false
+                    nonRoot: true
+                    alignCPUs: false
+                    enableManagedTenantQuota: false
+                    primaryUserDefinedNetworkBinding: false
+                    deployVmConsoleProxy: false
+                    persistentReservation: false
+                    autoResourceLimits: false
+                    deployKubevirtIpamController: false
+                  workloadUpdateStrategy:
+                    batchEvictionInterval: 1m0s
+                    batchEvictionSize: 10
+                    workloadUpdateMethods:
+                      - LiveMigrate
+                  uninstallStrategy: BlockUninstallIfWorkloadsExist
+                  resourceRequirements:
+                    vmiCPUAllocationRatio: 10
+# Fix our KVM_EMULATION setting on the Subscription
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: operators.coreos.com/v1alpha1
+                kind: ClusterServiceVersion
+                metadata:
+                  name: kubevirt-hyperconverged-operator.v4.17.3
+                  namespace: openshift-cnv
+                spec:
+                  install:
+                    spec:  
+                      deployments:
+                        - name: hco-operator
+                          spec:
+                            template:  
+                              spec:
+                                containers:
+                                  - name: hyperconverged-cluster-operator
+                                    env:
+                                    - name: KVM_EMULATION
+                                      value: 'false'
```

Comment:
I believe this can be specified in the Subscription: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md#env. It is not well documented ([there is an issue to improve it](https://issues.redhat.com/browse/ACM-20168)), but you can specify that `config` parameter in the OperatorPolicy subscription section.

Reply:
Customer won't actually want this function, I was using it to get around not having a virtualized SNO cluster, but Azure solved that problem for me.